ote/src/clj/ote/services/transit_visualization.sql

```diff
@@ -171,10 +171,24 @@ SELECT p.id, p.created,
  WHERE p."transport-service-id" = :service-id AND p."deleted?" = FALSE
  ORDER BY p.created DESC;
 
--- name: detected-changed-to-service-by-date
+-- name: detected-service-change-by-date
 SELECT c.date, c."added-routes", c."removed-routes", c."changed-routes", c."no-traffic-routes", c."current-week-date",
        c."different-week-date", c."change-date", c.created, c."transport-service-id"
 FROM "gtfs-transit-changes" c
 WHERE c."transport-service-id" = :service-id
   AND c.date = :date::DATE;
 
+-- name: detected-route-changes-by-date
```

Comment:
This could have been written also using `specql`. Probably you're not motivated to change this anymore, but would there be benefits on using that instead of this? :)

Reply:
postgres ::DATE conversion